lib/graphql/client.ts

```diff
@@ -141,14 +150,16 @@ export class GraphQLClient {
    * 3. Resubscribe all previously active subscriptions
    */
   private handleDisconnection = async (): Promise<void> => {
-    try {
-      await this.auth.tokenManager.refresh();
-    } catch (error) {
-      Object.values(this.subscriptions).forEach(({ onError }) => {
-        if (typeof onError === "function") {
-          onError(error);
-        }
-      });
+    if (this.auth) {
+      try {
+        await this.auth.tokenManager.refresh();
+      } catch (error) {
+        Object.values(this.subscriptions).forEach(({ onError }) => {
+          if (typeof onError === "function") {
+            onError(error);
+          }
+        });
+      }
     }
```

Comment:
we could save one if here
```suggestion
    try {
      await this.auth?.tokenManager.refresh();
    } catch (error) {
      Object.values(this.subscriptions).forEach(({ onError }) => {
        if (typeof onError === "function") {
          onError(error);
        }
      });
    }
```

Reply:
I think it would be more readable with the if 🤔